### PR TITLE
Nictiz 1669: add Task examples

### PR DESCRIPTION
**Pull request by niekvangalen**

### CODE REVIEW

**Comment by NictizEduard** [comment]
@LuudSlagter  Could you please review our changes?

File: examples/hg-ReferralComposition-01.xml [hunk 1]
```diff
@@ -25,76 +25,86 @@
...
+        <section>
+            <extension url="http://nictiz.nl/fhir/StructureDefinition/ext-TextValue">
+                <valueString value="graag uw fysiotherapeutische diagnose en behandeling, indien na 3 maanden geen verbetering retour voor eventuele doorverwijzing naar orthopeed"/>
+            </extension>
+            <title value="Procedurevoorstel"/>
+            <code>
+                <coding>
+                    <system value="http://snomed.info/sct"/>
+                    <code value="240471000146100"/>
+                    <display value="vertaling toelichting op aard van verwijzing (waarneembare entiteit)"/>
```
Comment (LuudSlagter) [inline examples/hg-ReferralComposition-01.xml:84]: Use 'toelichting op aard van verwijzing (waarneembare entiteit)' or 'toelichting op aard van verwijzing' (I'd prefer the latter).

File: examples/hg-ReferralComposition-01.xml [hunk 2]
```diff
@@ -25,76 +25,86 @@
...
+        <section>
+            <extension url="http://nictiz.nl/fhir/StructureDefinition/ext-TextValue">
+                <valueString value="Mevr. is ook onder behandeling voor COPD Gold II."/>
+            </extension>
+            <title value="Verder van belang"/>
+            <code>
+                <coding>
+                    <system value="http://snomed.info/sct"/>
+                    <code value="159121000146100"/>
+                    <display value="toelichting op voorgesteld beleid (waarneembare entiteit)"/>
```
Comment (LuudSlagter) [inline examples/hg-ReferralComposition-01.xml:101]: Suggestion: just use 'toelichting op voorgesteld beleid'.

File: examples/hg-ReferralTask-02.xml [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+    <focus>
+        <reference value="ServiceRequest/hg-ReferralServiceRequest-02" />
+        <display value="Verwijsbrief" />       
+    </focus>
+    <for>
+        <reference value="Patient/hg-ReferralServiceRequest-02-Patient-01" />
+        <display value="Marc van XXX_Quigley" />
+    </for>
+    <authoredOn>
+        <dateTime value="2023-06-22"/>
```
Comment (LuudSlagter) [inline examples/hg-ReferralTask-02.xml:24]: This date should be aligned with the ServiceRequest and Composition.

File: examples/hg-ReferralTask-01.xml [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+    <focus>
+        <reference value="ServiceRequest/hg-ReferralServiceRequest-01" />
+        <display value="Verwijsbrief" />       
+    </focus>
+    <for>
+        <reference value="Patient/hg-ReferralServiceRequest-01-Patient-01" />
+        <display value="Marijne van XXX_Prins" />
+    </for>
+    <authoredOn>
+        <dateTime value="2023-06-22"/>
```
Comment (LuudSlagter) [inline examples/hg-ReferralTask-01.xml:24]: This date should be aligned with the ServiceRequest and Composition.

**Review by LuudSlagter — CHANGES_REQUESTED** [review]

File: examples/hg-ReferralTask-01.xml [hunk 5]
```diff
@@ -20,15 +20,10 @@
...
         <display value="Marijne van XXX_Prins" />
     </for>
-    <authoredOn>
-        <dateTime value="2023-06-22"/>
-    </authoredOn>
+    <authoredOn value="2022-12-01"/>
     <requester>
-        <agent>
             <reference value="PractitionerRole/hg-ReferralServiceRequest-01-PractitionerRole-01"/>
-            <type value="PractitionerRole"/>
```
Comment (LuudSlagter) [inline examples/hg-ReferralTask-01.xml]: This line should be restored again, in line with other references ;)

File: examples/hg-ReferralTask-02.xml [hunk 6]
```diff
@@ -20,15 +20,10 @@
...
         <display value="Marc van XXX_Quigley" />
     </for>
-    <authoredOn>
-        <dateTime value="2023-06-22"/>
-    </authoredOn>
+    <authoredOn value="2022-12-01"/>
     <requester>
-        <agent>
             <reference value="PractitionerRole/hg-ReferralServiceRequest-02-PractitionerRole-01"/>
-            <type value="PractitionerRole"/>
```
Comment (LuudSlagter) [inline examples/hg-ReferralTask-02.xml]: See other example.

File: examples/hg-ReferralTask-01.xml [hunk 7]
```diff
@@ -0,0 +1,32 @@
...
+        <reference value="ServiceRequest/hg-ReferralServiceRequest-01" />
+        <display value="Verwijsbrief" />       
+    </focus>
+    <for>
+        <reference value="Patient/hg-ReferralServiceRequest-01-Patient-01" />
+        <display value="Marijne van XXX_Prins" />
+    </for>
+    <authoredOn value="2022-12-01"/>
+    <requester>
+            <reference value="PractitionerRole/hg-ReferralServiceRequest-01-PractitionerRole-01"/>
```
Comment (LuudSlagter) [inline examples/hg-ReferralTask-01.xml:25]: Change indentation of these two lines.

File: examples/hg-ReferralTask-02.xml [hunk 8]
```diff
@@ -0,0 +1,32 @@
...
+        <reference value="ServiceRequest/hg-ReferralServiceRequest-02" />
+        <display value="Verwijsbrief" />       
+    </focus>
+    <for>
+        <reference value="Patient/hg-ReferralServiceRequest-02-Patient-01" />
+        <display value="Marc van XXX_Quigley" />
+    </for>
+    <authoredOn value="2022-12-01"/>
+    <requester>
+            <reference value="PractitionerRole/hg-ReferralServiceRequest-02-PractitionerRole-01"/>
```
Comment (LuudSlagter) [inline examples/hg-ReferralTask-02.xml:25]: Change indentation of these two lines.

File: examples/hg-ReferralTask-02.xml [hunk 9]
```diff
@@ -0,0 +1,32 @@
...
+    <status value="requested" />
+    <intent value="order" />
+    <code>
+        <coding>
+            <system value="http://snomed.info/sct" />
+            <code value="3457005" />
+            <display value="verwijzen van patiënt" />
+        </coding>
+    </code>
+    <focus>
```
Comment (LuudSlagter) [inline examples/hg-ReferralTask-02.xml:15]: Add `. type`.

File: examples/hg-ReferralTask-02.xml [hunk 10]
```diff
@@ -0,0 +1,32 @@
...
+        <reference value="ServiceRequest/hg-ReferralServiceRequest-02" />
+        <display value="Verwijsbrief" />       
+    </focus>
+    <for>
```
Comment (LuudSlagter) [inline examples/hg-ReferralTask-02.xml:19]: Add `. type`.

File: examples/hg-ReferralTask-02.xml [hunk 11]
```diff
@@ -0,0 +1,32 @@
...
+        <reference value="Patient/hg-ReferralServiceRequest-02-Patient-01" />
+        <display value="Marc van XXX_Quigley" />
+    </for>
+    <authoredOn value="2022-12-01"/>
+    <requester>
```
Comment (LuudSlagter) [inline examples/hg-ReferralTask-02.xml:24]: Add `. type`.

File: examples/hg-ReferralTask-01.xml [hunk 12]
```diff
@@ -0,0 +1,32 @@
...
+    <status value="requested" />
+    <intent value="order" />
+    <code>
+        <coding>
+            <system value="http://snomed.info/sct" />
+            <code value="3457005" />
+            <display value="verwijzen van patiënt" />
+        </coding>
+    </code>
+    <focus>
```
Comment (LuudSlagter) [inline examples/hg-ReferralTask-01.xml:15]: Add `. type`.

File: examples/hg-ReferralTask-01.xml [hunk 13]
```diff
@@ -0,0 +1,32 @@
...
+        <reference value="ServiceRequest/hg-ReferralServiceRequest-01" />
+        <display value="Verwijsbrief" />       
+    </focus>
+    <for>
```
Comment (LuudSlagter) [inline examples/hg-ReferralTask-01.xml:19]: Add `. type`.

File: examples/hg-ReferralTask-01.xml [hunk 14]
```diff
@@ -0,0 +1,32 @@
...
+        <reference value="Patient/hg-ReferralServiceRequest-01-Patient-01" />
+        <display value="Marijne van XXX_Prins" />
+    </for>
+    <authoredOn value="2022-12-01"/>
+    <requester>
```
Comment (LuudSlagter) [inline examples/hg-ReferralTask-01.xml:24]: Add `. type`.

**Review by LuudSlagter — CHANGES_REQUESTED** [review]

**Review by LuudSlagter — APPROVED** [review]

**Pull request merged**